### PR TITLE
Add a function to get memlock hard limit

### PR DESCRIPTION
**Pull request by cliping**
Signed-off-by: cliping <lcheng@redhat.com>

### CODE REVIEW

File: virttest/utils_libvirt/libvirt_memory.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+"""
+Virtualization test - utility functions for libvirt
+
+:copyright: 2021 Red Hat Inc.
```
Comment (Yingshun) [inline virttest/utils_libvirt/libvirt_memory.py:4]: Could you please add 'memory' to make this file more specific? 
BTW, please update '2021' to '2022'. Thanks!

**Review by Yingshun — COMMENTED** [review]
Others LGTM

File: virttest/utils_libvirt/libvirt_memory.py (hunk 1 above)
Comment (cliping) [reply virttest/utils_libvirt/libvirt_memory.py]: Updated, thanks.

**Review by Yingshun — APPROVED** [review]

**Review by dzhengfy — APPROVED** [review]
LGTM

**Pull request merged**